package.json

```diff
@@ -1,11 +1,17 @@
 {
   "name": "jStorage",
```

Comment:
We should still do it like that. But doing it like that, and have it work easily and correctly with npm requires one git repository per package. That begs questions about naming and location. Maybe a new organisation or move it into be a part of the StaticWeb-organisation on github.

Regarding naming jStorage might not be the best since there are conflicting packages that does the same. What can stuff be stored within? I have some suggestions:

- BureauJS (Byrå)
- UniversalStoreJS
- SimpleStore
- Mound